spark/src/main/scala/io/substrait/spark/logical/ToSubstraitRel.scala

```diff
@@ -378,8 +381,25 @@ class ToSubstraitRel extends AbstractLogicalPlanVisitor with Logging {
   private def buildLocalFileScan(fsRelation: HadoopFsRelation): relation.AbstractReadRel = {
     val namedStruct = toNamedStruct(fsRelation.schema)
 
-    val ff = new FileFormat.ParquetReadOptions {
-      override def toString: String = "csv" // TODO this is hardcoded at the moment
+    val format = fsRelation.fileFormat match {
+      case _: CSVFileFormat =>
+        new FileFormat.DelimiterSeparatedTextReadOptions {
+          // default values for options specified here:
+          // https://spark.apache.org/docs/latest/sql-data-sources-csv.html#data-source-option
+          override def getFieldDelimiter: String = fsRelation.options.getOrElse("delimiter", ",")
+          override def getMaxLineSize: Long =
```

Comment:
same here

Reply:
Same as above, but I need to assign something, so I'll use the proto default value of `0`.